pom.xml

```diff
@@ -760,29 +772,39 @@ THE SOFTWARE.
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>jakarta-mail-api</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Moving this above the `mock-javamail` dependency for a few reasons:

- It comes first in group ID order, which is how Spotless would sort it.
- There was previously a requirement that `mock-javamail` come first, but that is no longer needed after implementing the system properties described in https://github.com/jenkinsci/lib-mock-javamail/pull/53.